### PR TITLE
Events API Send-RSVP support

### PR DESCRIPTION
**Pull request by yifanplanet**
- Events API Send-RSVP support

# License

I confirm that this contribution is made under the terms of the MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: src/main/kotlin/com/nylas/models/SendRSVPQueryParams.kt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ */
+enum class RsvpStatus(val value: String) {
+  YES("yes"),
+  NO("no"),
+  MAYBE("maybe");
+
+  override fun toString(): String {
+    return value
+  }
+}
```
Comment (mrashed-dev) [inline src/main/kotlin/com/nylas/models/SendRSVPQueryParams.kt:16]: Can we move this to its own file?

Comment (mrashed-dev) [inline src/main/kotlin/com/nylas/models/SendRSVPQueryParams.kt:16]: Can we move this to its own file?

File: src/main/kotlin/com/nylas/resources/Events.kt [hunk 2]
```diff
@@ -80,4 +80,21 @@ class Events(client: NylasClient) : Resource<Event>(client, Event::class.java) {
...
+  /**
+   * Send RSVP. Allows users to respond to events they have been added to as an attendee.
+   * @param identifier The identifier of the grant to act upon
+   * @param eventId The id of the Event to update.
+   * @param requestBody The values to send the RSVP with
+   * @param queryParams The query parameters to include in the request
+   * @return The send-rsvp response
+   */
+  @Throws(NylasApiError::class, NylasSdkTimeoutError::class)
+  fun sendRsvp(identifier: String, requestBody: SendRsvpRequest, queryParams: SendRsvpQueryParams): Response<Event> {
```
Comment (mrashed-dev) [inline src/main/kotlin/com/nylas/resources/Events.kt:93]: I believe the response type should be a model that represents a send rsvp response. [According to the docs](https://developer.nylas.com/docs/api/v3-beta/ecc/#post-/v3/grants/-grant_id-/events/-event_id-/send-rsvp) it should be an object like:

```
{
    request_id: string
}
```

**Review by mrashed-dev — CHANGES_REQUESTED** [review]
Left a couple of comments. Also if you can rename the files to match the casing of the class (i.e. instead of SendRSVPQueryParams it should be SendRsvpQueryParams). Thanks!

File: src/main/kotlin/com/nylas/resources/Events.kt (hunk 2 above)
Comment (yifanplanet) [reply src/main/kotlin/com/nylas/resources/Events.kt:93]: 👍

File: src/main/kotlin/com/nylas/models/SendRSVPQueryParams.kt (hunk 1 above)
Comment (yifanplanet) [reply src/main/kotlin/com/nylas/models/SendRSVPQueryParams.kt:16]: 👍

**Comment by yifanplanet** [comment]
@mrashed-dev Thanks for the review! The changes are made based on the request.

The `DeleteResponse` could be named into something more generic in the future.

**Review by mrashed-dev — APPROVED** [review]
🚀 Thank you @yifanplanet !

**Pull request merged**